Management/DomainModels/Comment.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Management.DomainModels
+{
+    public class Comment
+    {
+        public Location Location { get; }
+        public UserId UserId { get; }
+        public string CommentStr { get; }
```

Comment:
Have a question about this.
Why wrapping this with `TaggedString` can prevent Null exception?
My understanding is that I need to define another class `CommentString` that inherits from `TaggedString<CommentString>`, then the type here will be `CommentString` instead of `string`.
However, I think it's still possible for `CommentString` to be null, so we still need to have null check here for `CommentStr`?

Reply:
If it is the string, say var mengwen = "test".

You can then map variable mengwen to anyone.
For example, we have another class and one of the property is: public string Muzhi {get;set;}

Then you can just do Muzhi = mengwen; and you can assign null to Muzhi directly since sting is nullable.

But if we wrapped it with TaggedString, it will become a class. When you want to assign it in above example, you have to do Muzhi = mengwen.Value. It forces you to think about if there is a value.

In my next PR I am also going to add more restriction on tagged string class which will not allow we passed null or empty string to the ctor. I did not do it yet for fast prototype development purpose.

Make sense? If not you can remain as it is, for it's not logically wrong.

Reply:
Kindof make sense. Will keep using a string for this PR. I can update it latter if needed.